SCons/Tool/JavaCommon.py

```diff
@@ -88,7 +88,7 @@
     #     the multi-line comment begin and end tokens /* and */;
     #     array declarations "[]".
     _reToken = re.compile(r'(\n|\\\\|//|\\[\'"]|[\'"{\};.()]|' +
-                          r'\d*\.\d*|[A-Za-z_][\w$.]*|<[A-Za-z_]\w+>|' +
+                          r'\d*\.\d*|[A-Za-z_][\w$.]*|->|<[A-Za-z_]\w+>|' +
```

Comment:
Please update the comment block above with the change

Reply:
updated in 1b2e8e9